preupg/xmlgen/xml_utils.py

```diff
@@ -309,11 +306,11 @@ def create_xml_from_ini(self, main):
         }
         for key in self.keys:
             if 'check_script' not in key:
-                raise MissingTagsIniFileError
+                raise MissingTagsIniFileError(tags="check_script", ini_file=main)
```

Comment:
Note: Here is the place, where missing 'check_script' and 'solution' tags are detected. (my comment about  above content_title)?